index.html

```diff
@@ -215,6 +215,11 @@
       </p>
       <p>
         The <code><a href=
+        "http://www.w3.org/TR/WebIDL-1/#idl-ArrayBuffer"><dfn>ArrayBuffer</dfn></a></code>
```

Comment:
You can combine these if you like:

The BufferSource and ArrayBuffer are defined in [...]